src/resources/schema/document-code.yml

```diff
@@ -32,7 +32,10 @@
       Code linking is currently implemented only for the knitr engine 
       (via the [downlit](https://downlit.r-lib.org/) package). 
       A limitation of downlit currently prevents code linking 
-      if `code-line-numbers` is also `true`.
+      if `code-line-numbers` is also `true` and proper display
+      of code annotations if `code-annotations` is also set to `true`.
+      For `code-line-numbers` and `code-annotations` to work properly,
+      it is recommended to set `code-link` to `false`.
```

Comment:
I believe we should indeed add the limitation in there for `code-link` description. The limitation with `code-annotations` is missing. Basically the limitation is: 

`code-link: true` has no effect when `code-line-numbers` or `code-annotations` features are used.

Reply:
I reworded a bit the limitation sentence, to make it simpler and more accurate.

9f77c75b3911375c62e20843ba16007a00b97321